components/runners/ambient-runner/ambient_runner/platform/auth.py

```
@@ -384,6 +409,19 @@ def clear_runtime_credentials() -> None:
         os.environ.pop(key, None)
         cleared.append(key)
 
+    # Remove Gerrit config and gitcookies files
+    gerrit_config_path = os.environ.pop("GERRIT_CONFIG_PATH", None)
+    if gerrit_config_path:
+        cleared.append("GERRIT_CONFIG_PATH")
+    gerrit_dir = Path("/tmp/gerrit-mcp")
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/gerrit-mcp"

(S108)